Update description of source_module_id [TF-38184] (#2616)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/api-docs/explorer.mdx

| `providers`                       | `string`   | A comma separated list of providers used in this workspace. |
| `resources_drifted`               | `number`   | The count of resources that drift was detected for. |
| `resources_undrifted`             | `number`   | The count of resources that drift was not detected for. |

> | `source_module_id`                | `string`   | For workspaces using no-code modules, the identifier of the module. |

Reviewer: ```suggestion
| `source_module_id`                | `string`   | The source module identifier. Only available for workspaces created from a no-code module. |
```
